tcms_api/xmlrpc.py

```diff
@@ -103,11 +101,12 @@ def __init__(self, username, password, url):
             raise Exception("Unrecognized URL scheme")
 
         self._transport.cookiejar = CookieJar()
-        self.server = ServerProxy(
+        self.server = xmlrpc.client.ServerProxy(
             url,
             transport=self._transport,
             verbose=VERBOSE,
-            allow_none=1
+            allow_none=1,
+            context=ssl._create_unverified_context()

```

Comment:
can you explain what is happening here? What is your use case and why do we need this? Also what's the default in Python if you know? 

PS: this is going to need a test.